geode-assembly/src/test/java/org/apache/geode/management/internal/cli/commands/LogsAreFullyRedactedAcceptanceTest.java

```diff
@@ -64,54 +65,49 @@
   @Rule
   public GfshRule gfsh = new GfshRule();
 
-  @BeforeClass
-  public static void addSecuritySystemProperties() {
-    System.setProperty(gfshSysEnvUsername, gfshSysEnvUsernameValue);
-    System.setProperty(gfshSysEnvPassword, gfshSysEnvPasswordValue);
-    System.setProperty(gfshSysEnvUsernamePost, gfshSysEnvUsernameValuePost);
-    System.setProperty(gfshSysEnvPasswordPost, gfshSysEnvPasswordValuePost);
-  }
+  public LogsAreFullyRedactedAcceptanceTest() {}
 
   @Before
-  public void createPropertyFiles() throws IOException {
+  public void createDirectoriesAndFiles() throws IOException {
     propertyFile = gfsh.getTemporaryFolder().newFile("geode.properties");
     securityPropertyFile = gfsh.getTemporaryFolder().newFile("security.properties");
 
     Properties properties = new Properties();
     properties.setProperty(LOG_LEVEL, "debug");
+    properties.setProperty(propertyFileUsername, propertyFileUsernameValue);
+    properties.setProperty(propertyFilePassword, propertyFilePasswordValue);
     try (FileOutputStream fileOutputStream = new FileOutputStream(propertyFile)) {
       properties.store(fileOutputStream, null);
     }
 
     Properties securityProperties = new Properties();
-    securityProperties.setProperty(SECURITY_MANAGER, SimpleSecurityManager.class.getName());
+    securityProperties.setProperty(SECURITY_MANAGER, ExampleSecurityManager.class.getName());
+    securityProperties.setProperty(ExampleSecurityManager.SECURITY_JSON, "security.json");
+    securityProperties.setProperty(securityPropertyFileUsername, securityPropertyFileUsernameValue);
+    securityProperties.setProperty(securityPropertyFilePassword, securityPropertyFilePasswordValue);
     try (FileOutputStream fileOutputStream = new FileOutputStream(securityPropertyFile)) {
       securityProperties.store(fileOutputStream, null);
     }
   }
 
-  @AfterClass
-  public static void removeSecuritySystemProperties() {
-    System.clearProperty(gfshSysEnvUsername);
-    System.clearProperty(gfshSysEnvPassword);
-    System.clearProperty(gfshSysEnvUsernamePost);
-    System.clearProperty(gfshSysEnvPasswordPost);
-  }
-
   @Test
   public void logsDoNotContainStringThatShouldBeRedacted() throws FileNotFoundException {
-    String startLocatorCmd = new CommandStringBuilder("start locator")
-        .addOption("properties-file", propertyFile.getAbsolutePath())
-        .addOption("security-properties-file", securityPropertyFile.getAbsolutePath())
-        .addOption("name", "test-locator").getCommandString();
+    // The json is in the root resource directory.
+    String securityJson = getClass().getResource("/").getFile();
+    String startLocatorCmd =
```

Comment:
I would also try to add the username and password in as --J options in the start locator/server command